integration.md

```diff
@@ -220,3 +220,4 @@ data-flow | The type of on-demand form to display. The supported values are `pur
 data-locale | The locale in which the form should be presented.
 data-amount | This attribute is valid only for purchase forms and denotes the purchase's total price. The value should be multiplied by 100, so if your price is "9,90" it should be sent as "990".
 data-currency | This attribute is valid only for purchase forms and denotes the purchase's currency, as an [ISO 4217 code](https://en.wikipedia.org/wiki/ISO_4217).
+data-merchant-context | This attribute is used to track conversion rate in Google Analytics based on any given attribute, i.e. "daily pass".
```

Comment:
I suggest writing that this parameter is optional (maybe another column) and we recommend sending information about the product. Klarna will use this to segment your traffic and will be able to offer ideas that may help your conversion as a merchant. A practical example is to divide the traffic between "daily pass" and "weekly pass".

And please talk to Tomer about it.